app/src/main/java/io/crate/node/CrateNode.java

```diff
@@ -46,6 +47,7 @@
         UDCPlugin.class,
         URLRepositoryPlugin.class,
         S3RepositoryPlugin.class,
+        S3CopyPlugin.class,
```

Comment:
Would it be possible to remove this concrete dependency and rely on the dynamic loading? See the todo:

https://github.com/crate/crate/blob/5e6ffad8bbf32bff9dd4d5e5ff85aa73803d5da5/app/build.gradle#L58-L59

The mechanism here is preferred:

https://github.com/crate/crate/blob/5e6ffad8bbf32bff9dd4d5e5ff85aa73803d5da5/app/build.gradle#L34-L43

Reply:
referred to #11307, as the PR explains, it does not work if ran as `app:run` but if I build and run locally, copy from s3 works fine.

Reply:
> it does not work if ran as app:run but if I build and run locally, copy from s3 works fine.

yep that's expected. `app:run` doesn't load the plugins.